spk/fengoffice/src/wizard/install_uifile.sh

```diff
@@ -200,7 +200,7 @@ PAGE_INSTALL_CONFIG=$(/bin/cat<<EOF
         "desc": "A 'fengoffice' MySQL user and database will be created. Please provide a password for the 'fengoffice' user.",
         "subitems": [{
             "key": "wizard_mysql_password_fengoffice",
-            "desc": "fengoffice password",
+            "desc": "User password",
```

Comment:
now it does not match the description above, that says it is the password for the `'fengoffice' user`

To make it clear I propose to adjust both subitem descriptions:

`MySQL 'root' password` - instead of `Root password`
and
`MySQL 'fengoffice' password` - instead of `fengoffice password` or `User password`

Reply:
I've updated the descriptions accordingly. If we are to standardise this, many other package wizards will need to be updated in the future.